docs/UserGuide.md

```diff
@@ -233,7 +232,7 @@ Examples:
 ## FAQ
 
```

Comment:
Do you mind adding a empty line between `deletelog -a` and the `----------`?
From the website version it appears that there's some formatting error.